contracts/RiskFund/RiskFund.sol

```diff
@@ -224,6 +224,11 @@ contract RiskFund is Ownable2StepUpgradeable, ExponentialNoError, ReserveHelpers
      * @return Number of swapped tokens.
      */
     function swapAllPoolsAssets() external returns (uint256) {
+        bool canSwapAllPoolsAsset = AccessControlManager(accessControl).isAllowedToCall(
+            msg.sender,
+            "swapAllPoolsAssets()"
+        );
+        require(canSwapAllPoolsAsset, "Risk fund: Not authorized to swap pool assets.");
```

Comment:
currently the definition of `swapPoolsAssets` is changed in [PR 114](https://github.com/VenusProtocol/isolated-pools/pull/114)